pynq/lib/logictools/pattern_generator.py

```diff
@@ -294,7 +297,38 @@ def setup(self, waveform_dict,
             self.logictools_controller.pin_map[i] = 'OUTPUT'
         for i in self.analysis_pins:
             self.logictools_controller.pin_map[i] = 'INPUT'
+            
+        # Write the initial value to the output pins
+        if(initial_value is not 0xFFFF):
```

Comment:
This is causing a lint error - would you be able to change the `is not` to `!=`?

Reply:
Updated